test/rubygems/test_gem_install_update_options.rb

```diff
@@ -151,14 +151,45 @@ def test_user_install_disabled_read_only
 
       Gem.use_paths @gemhome, @userhome
 
-      assert_raise(Gem::FilePermissionError) do
+      assert_raise(Gem::FilePermissionError, Errno::EACCES) do
```

Comment:
May no longer be needed after the latest changes?

Reply:
I'll check if this is still needed. Still not sure what caused it to be needed, honestly.

Reply:
Currently testing this now. Unfortunately, I can only reproduce it in CI, so it'll be about an hour before I have an update. 😅

Reply:
You were right, I was able to undo that change without issue. 👍